org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslation.java

```diff
@@ -0,0 +1,38 @@
+package ca.uwaterloo.watform.rapidDash;
+
+import ca.uwaterloo.watform.parser.DashModule;
+import edu.mit.csail.sdg.ast.Sig;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static edu.mit.csail.sdg.alloy4.TableView.clean;
+
+/**
+ * Mutable; this class represents an Dash to Python translation module
+ */
+
+public class DashPythonTranslation {
+
+    private DashModule dashModule;
+
+    public List<String> basicSigLabels;
+
+    /**
+     * Constructs a new DashPythonTranslation object
+     *
+     * @param dashModule - the DashModule we want to translate
+     */
+    public DashPythonTranslation(DashModule dashModule) {
+        this.dashModule = dashModule;
+        this.basicSigLabels = dashModule.sigs.values().stream()
+                .filter(this::isSubSig)
```

Comment:
Question: what is subsig?
I saw some explanation in code
```
    /**
     * Nonnull if this sig is a PrimSig and therefore not a SubsetSig.
     */
    public final Pos             isSubsig;
```
kinda confused

Reply:
I don't know, I also saw that explanation. I am just using this as the helper function name for now. Maybe later after we add more kinds of signatures, we could found a standardized way of naming all of them?

Reply:
Yes, I could rename it to isBasicSig, but the concept of basic signature is also kind of vague. I expect the code to evolve in later tickets.

Reply:
Makes sense.